test/unit/spec/v2/ion/i18nTransformer_spec.js

```diff
@@ -1752,4 +1753,128 @@ describe('v2/ion/i18nTransformer', function() {
     };
     expect(getI18NParams(remediation, 'google_otp')).toEqual([]);
   });
+
+  it('should get translated error message if the browser thrown error supported in our i18n bundle', () => {
+    expect(getMessageFromBrowserError({
+      message: 'browser message',
+      name: 'NotAllowedError'
+    })).toEqual('unit test - translated browser thrown error');
+  });
+
+  it('should get original message if the browser thrown error is not supported in our i18n bundle', () => {
+    expect(getMessageFromBrowserError({
+      message: 'browser message',
+      name: 'UnsupportedErrorName'
+    })).toEqual('browser message');
+  });
+
+  const expectedWebAuthnGenericError = 'You are currently unable to use a Security key or biometric authenticator. Try again.';
+
+  it.each([
+    [
+      'authfactor.webauthn.error.assertion_validation_failure',
+      {
+        message:'None of the WebAuthn authenticators the user has enrolled for was able to validate the assertion',
```

Comment:
qq @mpeng-okta @vincentvoong-okta  why dont we want to surface this error ? In case user had many yubikeys and was using the one he/she dint enroll and was wondering what happened wouldnt this be helpful ?

Reply:
Since we use `allowCredentials` anyway, if the user uses an unenrolled one we'd get a client error anyway I think:
https://github.com/okta/okta-signin-widget/blob/5689b077fd8f5b0ea14841cc94fa7e5dcc044187/src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js#L66-L75